app-frontend/src/app/pages/projects/edit/browse/browse.controller.js

```diff
@@ -4,10 +4,9 @@ const _ = require('lodash');
 export default class ProjectAddScenesBrowseController {
     constructor( // eslint-disable-line max-params
         $log, $state, $uibModal, $scope, $timeout, mapService, sceneService,
-        projectService, gridLayerService, sessionStorage
+        projectService, gridLayerService, sessionStorage, planetLabsService, authService
```

Comment:
this controller is getting quite massive - a lot of the logic (pretty much anything that isn't directly modifying the dom / view state) can be put into services.
For instance, functions that parse params / construct requests
We can meet to figure out what should go where.